.github/workflows/tests.yml

```
@@ -0,0 +1,35 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
```

actionlint (1.7.10)

[error] "runs-on" section is missing in job "test"

(syntax-check)